src/ray/raylet/node_manager.cc

```diff
@@ -1244,7 +1244,8 @@ void NodeManager::DeleteLocalURI(const std::string &uri, std::function<void(bool
     cb(true);
   }
 
-  auto from_path = resource_path / boost::filesystem::path(uri.substr(pos + sep.size()));
+  auto from_path =
+      resource_path / boost::filesystem::path(uri.substr(pos + sep.size())).stem();
```

Comment:
was this a bug?

Reply:
No, it's not. Previously, the file is "/path/hash.zip" now the directory is "/path/hash"